src/pages/workspaces/workspace/WorkspaceMenu.js

```diff
@@ -47,39 +47,49 @@ const DynamicWorkspaceMenuContent = ({ namespace, name, onClone, onShare, onDele
   })
 }
 
-const WorkspaceMenuContent = ({ canShare, isAzureWorkspace, isLocked, isOwner, onClone, onShare, onLock, onDelete, workspaceLoaded = true }) => {
+export const tooltipText = {
```

Comment:
@yunip20 I believe our convention is to use lowercase initial letters. I moved all the tooltip text options into this object so we could fully test everything.